Core/GameEngine/Include/Common/GameDefines.h

```diff
@@ -0,0 +1,30 @@
+/*
+**	Command & Conquer Generals Zero Hour(tm)
+**	Copyright 2025 TheSuperHackers
+**
+**	This program is free software: you can redistribute it and/or modify
+**	it under the terms of the GNU General Public License as published by
+**	the Free Software Foundation, either version 3 of the License, or
+**	(at your option) any later version.
+**
+**	This program is distributed in the hope that it will be useful,
+**	but WITHOUT ANY WARRANTY; without even the implied warranty of
+**	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+**	GNU General Public License for more details.
+**
+**	You should have received a copy of the GNU General Public License
+**	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// Note: Retail compatibility must not be broken before this project officially does.
+// Use RETAIL_COMPATIBLE_XFER_CRC and RETAIL_COMPATIBLE_XFER_SAVE to guard breaking changes.
+
+#ifndef RETAIL_COMPATIBLE_XFER_CRC
+#define RETAIL_COMPATIBLE_XFER_CRC (1) // Game is expected to be Xfer CRC compatible with retail Generals 1.08, Zero Hour 1.04
```

Comment:
Rename to `RETAIL_COMPATIBLE_CRC` because more things concern CRC than just Xfer.

Reply:
Fixed.